commit 6fa15cb04eb917c5710813473171640508a03cd4 (refs/changes/71/4571/1)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T18:29:34+00:00 (1 year, 2 months ago)
    
    TRITON-606 ur-agent should be aware of alternate admin nictags (bump version)

diff --git a/package.json b/package.json
index 371c404..2daffec 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdc-ur-agent",
   "description": "Ur",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "author": "Joyent (joyent.com)",
   "main": "./ur-agent",
   "private": true,
